pkg/setting/setting.go

```diff
@@ -475,6 +475,8 @@ type Cfg struct {
 	// RBAC single organization. This configuration option is subject to change.
 	RBACSingleOrganization bool
 
+	Zanzana ZanzanaSettings
```

Comment:
I intentionally left out changes to default.ini and sample.ini for now

Reply:
That makes sense since it's highly experimental now.